### PR TITLE
Feature/204 notification

### PR DESCRIPTION
**Pull request by mkowalczyk97**
Added notification feature.

You can see test notifications scheduled for next 10 seconds in Notes and Weather component.

Configuration of notifications is in NotificationManager.js.

Cheers

### CODE REVIEW

**Review by weronikaolejniczak — CHANGES_REQUESTED** [review]
Add google-services.json to .gitignore and change API key.

File: src/domains/weather/containers/Weather.js [hunk 1]
```diff
@@ -25,6 +26,7 @@ const Weather = (props) => {
     state.trips.availableTrips.find((item) => item.id === tripId),
   );
   const region = selectedTrip.region;
+  console.log(region)
```
Comment (CalderBriar) [inline src/domains/weather/containers/Weather.js:29]: I think it would be better to comment or delete non-vital console.log

**Review by CalderBriar — COMMENTED** [review]
One no longer necessary console.log. Just do provide the requested change of api key and add the json file to gitignore please.

**Comment by weronikaolejniczak** [comment]
That's okay for now. 
google-services.json can be downloaded from Firebase console.

**Review by weronikaolejniczak — APPROVED** [review]

**Pull request merged**